CMakeLists.txt

```
@@ -545,7 +545,7 @@ message(STATUS "Using Lua interpreter: ${LUA_PRG}")
 
 option(COMPILE_LUA "Pre-compile Lua sources into bytecode (for sources that are included in the binary)" ON)
```

`DEBUG` is set to `0` or `1` above at lines 126 to 130.
```suggestion
if(DEBUG)
  option(COMPILE_LUA "Pre-compile Lua sources into bytecode (for sources that are included in the binary)" OFF)
else()
  option(COMPILE_LUA "Pre-compile Lua sources into bytecode (for sources that are included in the binary)" ON)
endif()
```

---

Thanks I had actually tried something similar earlier today but it didn't work for some reason back then . This works fine .